### PR TITLE
Add VMware support

### PR DESCRIPTION
**Pull request by FedericoCeratto**
Initial version, to be merged with https://github.com/kubic-project/automation/pull/608

### CODE REVIEW

File: vars/cloneAllKubicRepos.groovy [hunk 1]
```diff
@@ -22,7 +22,8 @@ def call(Map parameters = [:]) {
 
     timeout(5) {
         parallel 'automation': {
-            cloneKubicRepo(gitBase: gitBase, branch: branch, credentialsId: credentialsId, ignorePullRequest: ignorePullRequest, repo: "automation")
+        // FIXME
```
Comment (ereslibre) [inline vars/cloneAllKubicRepos.groovy]: Can we be more specific about what needs fixing here?

File: vars/configureEnvironment.groovy [hunk 2]
```diff
@@ -35,7 +35,8 @@ Environment call(Map parameters = [:]) {
     timeout(90) {
         try {
             dir('automation/velum-bootstrap') {
-                if (chooseCrio) {
+                if (false) {
+                //if (chooseCrio) {
```
Comment (ereslibre) [inline vars/configureEnvironment.groovy]: I get this is tricky, can we remove the block altogether or do something slightly different? If we plan to fix this right afterwards in a followup PR it's fine for me then.

File: vars/cloneAllKubicRepos.groovy (hunk 1 above)
Comment (FedericoCeratto) [reply vars/cloneAllKubicRepos.groovy]: It's needed to test the PR and removed before merge.

File: vars/configureEnvironment.groovy (hunk 2 above)
Comment (FedericoCeratto) [reply vars/configureEnvironment.groovy]: Used temporarily during testing - removed.

**Pull request merged**